apps/studio/src/features/editing-experience/components/EditLinkPreview.tsx

```diff
@@ -0,0 +1,80 @@
+import type { IsomerSitemap } from "@opengovsg/isomer-components"
+import { useAtomValue } from "jotai"
+
+import { useQueryParse } from "~/hooks/useQueryParse"
+import { editLinkSchema } from "~/pages/sites/[siteId]/links/[linkId]"
+import { trpc } from "~/utils/trpc"
+import { linkAtom } from "../atoms"
+import PreviewWithoutSitemap from "./PreviewWithoutSitemap"
+import { ViewportContainer } from "./ViewportContainer"
+
+export const EditCollectionLinkPreview = (): JSX.Element => {
+  const { description: summary, date } = useAtomValue(linkAtom)
+  const { linkId, siteId } = useQueryParse(editLinkSchema)
+  const [permalink] = trpc.page.getFullPermalink.useSuspenseQuery(
+    {
+      pageId: linkId,
+      siteId,
+    },
+    { refetchOnWindowFocus: false },
+  )
+
+  const [{ parent }] = trpc.resource.getParentOf.useSuspenseQuery({
+    resourceId: String(linkId),
+    siteId,
+  })
+
+  const [children] = trpc.collection.getSiblingsOf.useSuspenseQuery({
+    siteId,
+    resourceId: linkId,
+  })
```

Comment:
issue: do we need to actually show all the siblings of the collection? I thought we were only intending to show only one, which is the exact item that the user is editing.

My concern is that due to the way we are sorting (type then title, from the trpc procedure), we cannot guarantee that the actual item that the user is editing will be visible in the preview.

My suggestion is to just skip showing of the siblings here when editing a specific link, and only show that single item within the context of the collection.

Reply:
removed!